letsencrypt-auto-source/pieces/fetch.py

```diff
@@ -112,6 +120,14 @@ def verified_new_le_auto(get, tag, temp_dir):
                             "certbot-auto.", exc)
 
 
+def create_CERT_NONE_context():
```

Comment:
For consistency with the noun-like names of other functions that have return values, this would be better named "cert_none_context".